lessons/04-structured-output/README.md

```diff
@@ -0,0 +1,590 @@
+# Structured output
```

Comment:
Really like the story, game, and overall flow. Nice job integrating the conversation throughout. Aside from my comments, my main suggestion would be running the main story through AI again and trying to shorten it even more. We have to create a nice balance of fun story with the time it takes people to read it. It's a little on the long side IMO (but it's good and fun :-)).